homeassistant/components/rhasspy/const.py

```diff
@@ -0,0 +1,122 @@
+"""Constants for Rhasspy integration.
+
+For more details about this integration, please refer to the documentation at
+https://home-assistant.io/integrations/rhasspy/
+"""
+DOMAIN = "rhasspy"
+
+# Language-specific profiles available for Rhasspy.
+# See https://github.com/synesthesiam/rhasspy-profiles/releases
+SUPPORT_LANGUAGES = [
+    "en-US",
+    "nl-NL",
+    "fr-FR",
+    "de-DE",
+    "el-GR",
+    "it-IT",
+    "pt-BR",
+    "ru-RU",
+    "es-ES",
+    "sv-SV",
+    "vi-VI",
+]
+
+# -----------------------------------------------------------------------------
+# Rhasspy Intents
+# -----------------------------------------------------------------------------
+
+# Confirms or disconfirms if a device is currently on
+INTENT_IS_DEVICE_ON = "IsDeviceOn"
```

Comment:
These could be added to the Intent integration. We should make sure that all intents that are added to Home Assistant start with `Hass`